PulseEco/AppRoot/MainView.swift

```diff
@@ -55,6 +56,11 @@ struct MainView: View {
                 })
             }
         }
+        .onAppear() {
+            if (locationManager.isAuthorizedOrNotDetermined()) {
+                locationManager.requestLocation()
+            }
+        }
     }
```

Comment:
we can just trigger requestWhenInUseAuthorization without any checks since it will not show the prompt if we already if we previously granted or denied access